GSE50409.Rmd

```diff
@@ -13,4 +12,43 @@ output: html_document
 - Domas Motiejūnas
 
 # Analysis
-
+```{r setup, echo=FALSE, include=FALSE}
+library(GEOquery)
+```
+- Automatically download the data from GEO
+```{r echo=FALSE, include=FALSE}
+gse50409 <- getGEO("gse50409", destdir="./")
+```
+- Obtain the matrix of beta values where each row corresponds to probes and each column corresponds to samples
+```{r}
+matrix <- exprs(gse50409[[1]])
+matrix[1:20, 1:20]
+```
+- How many samples and how many probes do you have in the data?
+```{r}
+probes <- featureNames(featureData(gse50409[[1]]))
+length(probes)
+samplesNames <- sampleNames(phenoData(gse50409[[1]]))
+length(samplesNames)
+```
+- How are the beta values distributed?
+```{r out.width = '100%'}
+boxplot(matrix, outline = FALSE, boxwex = 0.7, notch = T, las = 2, main = "GSE50409")
```

Comment:
boxplot geras tada, kai norim ivertinti kiekvieno meginio beta verciu pasiskirstyma atskirai, bet jei mus domina viso dataseto bendras pasiskirstymas, ji sunku ivertinti is boxploto, tam geriau naudoti [kitus grafikus](https://www.google.lt/search?rlz=1C5CHFA_enLT765LT767&ei=WLCfWtvdMeOagAaWuaf4DQ&q=plotting+distribution+in+r&oq=plotting+distribution+in+r&gs_l=psy-ab.3..0l5j0i22i30k1l5.1446.1788.0.1877.5.5.0.0.0.0.135.462.1j3.4.0....0...1c.1.64.psy-ab..1.4.461....0.otmenspT5XM).